kpi/deployment_backends/kobocat_backend.py

```diff
@@ -865,7 +865,7 @@ def bulk_update_submissions(
                 method='POST', url=self.submission_url, files=files
             )
             kc_response = self.__kobocat_proxy_request(
-                kc_request, user=self.asset.owner
+                kc_request, user=requesting_user
```

Comment:
I think I found why you chose to use `self.asset_owner` at the first place. 
`requesting_user` needs `add_submissions` to be allowed to duplicate in KC, otherwise it returns a 403. 
Sending the request with the owner bypasses this validation. 

This brings up this issue: https://github.com/kobotoolbox/kpi/issues/3148

Would you mind to change the tests?

And if you are willing, would you mind to change the front end too? 
i.e. Show the "duplicate" button only to users who have both permissions

Otherwise, push it to one of our front-end friends.

While looking at the front-end code, we should fix this typo. 
![Screen Shot 2021-05-07 at 11 14 24](https://user-images.githubusercontent.com/24226825/117473451-fdb4c500-af27-11eb-83d8-8d665c3e1485.png)

Reply:
- [ ] change back to `self.asset.owner` (also for bulk edit)
- [ ] update tests
- [ ] change permissions to required both `add_submissions` and `change_submissions` otherwise `403`
- [ ] update frontend to only show `DUPLICATE` button when user has both permissions
- [ ] fix typos

Reply:
> change back to self.asset.owner (also for bulk edit)
No, we want to keep `requesting_user`

Reply:
You're right 🤦 I corrected that point with what I actually did:

> change back to self.asset.owner for bulk edit